test/e2e/alphagenerate/generate_test.go

```diff
@@ -191,4 +204,11 @@ func ReGenerateProject(kbc *utils.TestContext) {
 		filepath.Join(kbc.Dir, "testdir2", "PROJECT"), grafanaPlugin)
 	ExpectWithOffset(1, err).NotTo(HaveOccurred())
 	ExpectWithOffset(1, fileContainsExpr).To(BeTrue())
+
+	By("checking if the generated grafana config file has the same content as the old one")
+	cmp := equalfile.New(nil, equalfile.Options{}) // compare using single mode
+	equal, err := cmp.CompareFile(filepath.Join(kbc.Dir, "testdir2", "grafana/custom-metrics/config.yaml"),
+		filepath.Join(kbc.Dir, "grafana/custom-metrics/config.yaml"))
+	ExpectWithOffset(1, err).NotTo(HaveOccurred())
```

Comment:
Good idea.  Acutally I'm still a little confused when should use this one. I think 'ExpectWithOffset' could provide the message about the 'call stack', so it will be used when in a function(maybe a layer of functions). And if not the case, we should use 'Expect'. Is it right?